backend/models/user_model.py

```diff
@@ -23,19 +13,17 @@ class User(Document):
     last_name = StringField(max_length=50, default="No_Last")
     email = EmailField(required=True)
     password = StringField(required=True, min_length=6)
-    phone = IntField(min_value=10000000, max_value=99999999)
+    phone = StringField(max_length=11, regex=r"\d")
```

Comment:
i think in the figma the prefix is +373 which leaves 8 digits to be added, so the max_length should be 8 not 11, we talked about this with Vitalie in the project slack channel